### PR TITLE
Fix #13: Add golang to the database

### LINKED ISSUE #13: Add Go
See `Android` in README.md to get an idea of the contents of the PR for this issue.

### PR DESCRIPTION
**Pull request by jonathanalvares9009**
## Explanation

This PR adds information regarding Golang Programming Language.
It fixes #13 
## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR does not break any part of the existing code.
- [x] The PR creator have carefully read & adher to the contributing guidelines of the project.
- [x] The PR is made to the branch that's **called** "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "main" and not on any other branch.
- [x] Have you followed the guidelines in the Contributing document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/CSwala/awesome-computer-science/pulls) for the same tech?
- [x] Is your submission is according to the desired pattern ?
- [x] Have you placed the contents in the `Readme.md` keeping the sorting order in mind ?

Does you submission include -

- [x]	Category/Type
- [x]	Official Documnetation  
- [x]	Best YouTube creators  
- [x]	Useful Websites  
- [x]	Dedicated Github page 
- [x]	Best Courses  
- [x]	Ohter Tips / Hacks

### CODE REVIEW

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Find more useful websites and other content like GH pages as GOlang has a lot of content

**Comment by jonathanalvares9009** [comment]
@riturajjain2000 added more info

File: Dictionary.json [hunk 1]
```diff
@@ -283,6 +283,41 @@
...
+		"Website6": "https://builtin.com/software-engineering-perspectives/golang-advantages",
+		"Website7": "https://yourbasic.org/golang/advantages-over-java-python/",
+		"Website8": "https://www.freecodecamp.org/news/here-are-some-amazing-advantages-of-go-that-you-dont-hear-much-about-1af99de3b23a/",
+		"Dedicated GH page1" : "https://github.com/DaoCloud/golang-mongo-sample",
+		"Dedicated GH page2": "https://github.com/kyawmyintthein/golangRestfulAPISample",
+		"Dedicated GH page3": "https://github.com/digitalocean/sample-golang",
+		"Dedicated GH page4": "https://github.com/golang/go",
+		"Dedicated GH page5": "https://github.com/golang/oauth2",
+		"Dedicated GH page6": "https://github.com/golang/gofrontend",
+		"Dedicated GH page7": "https://github.com/golang/appengine",
```
Comment (riturajjain2000) [inline Dictionary.json:308]: https://github.com/avelino/awesome-go
add awesme pages also
search for awesome golang on GH and add them,

**Review by riturajjain2000 — CHANGES_REQUESTED** [review]
Add awesome pages , as they are the best in class from the community

**Review by riturajjain2000 — APPROVED** [review]

**Pull request merged**